<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Login_page">
       <property name="styleSheet">
        <string notr="true">*{
	font-family:century gothic;
	font-size:24px;
}

QFrame{
	background:#333;
	border-radius:15px;
}

QPushButton{
	color:white;
	background:#f02f00;
	border-radius:40px;
}

QToolButton{
	background:white;
	border-radius:60px;
}

QLabel{
	color:white;
}

QPushButton:hover{
	color:white;
	border-radius:40px;
	background:#B02300;

}

QPushButton:pressed{
	background:#701600;
}

QLineEdit{
	background:transparent;
	border:none;
	color:#717072;
	border-bottom:1px solid #717072;
}</string>
       </property>
       <widget class="QToolButton" name="toolButton">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>20</y>
          <width>120</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../Images/icon2.png</normaloff>../Images/icon2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
       </widget>
       <widget class="QFrame" name="LoginFrame">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>710</width>
          <height>730</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="WelcomeLabel">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>70</y>
           <width>221</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Welcome, Guest</string>
         </property>
        </widget>
        <widget class="QPushButton" name="LoginBtn">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>530</y>
           <width>671</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>Log in</string>
         </property>
        </widget>
        <widget class="QPushButton" name="RegisterBtn">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>630</y>
           <width>671</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="UserNameLineEdit">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>170</y>
           <width>571</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>8</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="placeholderText">
          <string>Username</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="PassLineEdit">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>410</y>
           <width>571</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>10</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="IdLineEdit">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>290</y>
           <width>571</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>9</number>
         </property>
         <property name="placeholderText">
          <string>Id</string>
         </property>
        </widget>
        <widget class="QLabel" name="UserLabel">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>130</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
        <widget class="QLabel" name="IdLabel">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>250</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Id:</string>
         </property>
        </widget>
        <widget class="QLabel" name="PasswordLabel">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>370</y>
           <width>181</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Password:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
        <widget class="QFrame" name="frame_window_navigation">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>10</y>
           <width>100</width>
           <height>38</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
 	border-radius:8px
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_minimise">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../Images/minus-7-128.png</normaloff>../Images/minus-7-128.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_close">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../Images/x-mark-24.png</normaloff>../Images/x-mark-24.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <zorder>LoginFrame</zorder>
       <zorder>toolButton</zorder>
      </widget>
      <widget class="QWidget" name="Register_page">
       <property name="styleSheet">
        <string notr="true">*{
	font-family:century gothic;
	font-size:24px;
}

QFrame{
	background:#333;
	border-radius:15px;
}

QPushButton{
	color:white;
	background:#f02f00;
	border-radius:40px;
}

QToolButton{
	background:white;
	border-radius:60px;
}

QLabel{
	color:white;
}

QPushButton:hover{
	color:white;
	border-radius:40px;
	background:#B02300;

}

QLineEdit{
	background:transparent;
	border:none;
	color:#717072;
	border-bottom:1px solid #717072;
}</string>
       </property>
       <widget class="QToolButton" name="toolButton_2">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>20</y>
          <width>120</width>
          <height>120</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../Images/icon2.png</normaloff>../Images/icon2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
       </widget>
       <widget class="QFrame" name="RegisterFramer">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>80</y>
          <width>710</width>
          <height>841</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="RegisterLabel">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>70</y>
           <width>221</width>
           <height>61</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Register Form</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QPushButton" name="CancelBtn">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>750</y>
           <width>671</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
        <widget class="QPushButton" name="RegisterBtn_2">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>650</y>
           <width>671</width>
           <height>81</height>
          </rect>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="UserNameLineEdit_2">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>260</y>
           <width>570</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>8</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="placeholderText">
          <string>Your Username</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="PassLineEdit_2">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>460</y>
           <width>570</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>10</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Your Password</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="IdLineEdit_2">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>350</y>
           <width>570</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>9</number>
         </property>
         <property name="placeholderText">
          <string>Your Id</string>
         </property>
        </widget>
        <widget class="QLabel" name="UserLabel_2">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>220</y>
           <width>571</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Your Username:</string>
         </property>
        </widget>
        <widget class="QLabel" name="IdLabel_2">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>320</y>
           <width>570</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Your Id:</string>
         </property>
        </widget>
        <widget class="QLabel" name="PasswordLabel_2">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>420</y>
           <width>570</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Your Password:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="ConfirmPassLineEdit">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>570</y>
           <width>571</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>10</number>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Confirm Password</string>
         </property>
        </widget>
        <widget class="QLabel" name="ConfirmPassLabel">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>530</y>
           <width>571</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Confirm Password:</string>
         </property>
         <property name="textFormat">
          <enum>Qt::PlainText</enum>
         </property>
        </widget>
        <widget class="QLineEdit" name="FullNameLineEdit">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>160</y>
           <width>570</width>
           <height>51</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="maxLength">
          <number>50</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="placeholderText">
          <string>Your Full Name</string>
         </property>
        </widget>
        <widget class="QLabel" name="UserLabel_3">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>120</y>
           <width>570</width>
           <height>41</height>
          </rect>
         </property>
         <property name="text">
          <string>Your Full Name:</string>
         </property>
        </widget>
        <widget class="QFrame" name="frame_window_navigation_2">
         <property name="geometry">
          <rect>
           <x>600</x>
           <y>10</y>
           <width>100</width>
           <height>38</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton{
 	border-radius:8px
}
</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btn_minimise_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../Images/minus-7-128.png</normaloff>../Images/minus-7-128.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_close_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../Images/x-mark-24.png</normaloff>../Images/x-mark-24.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <zorder>RegisterFramer</zorder>
       <zorder>toolButton_2</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
